api/jobs/rules.py

```diff
@@ -237,9 +238,10 @@ def create_jobs(db, container_before, container_after, container_type):
     spawned_jobs = []
 
     for pj in potential_jobs:
-        pj['job'].insert()
-        spawned_jobs.append(pj['rule']['alg'])
+        job_map = pj['job'].map()
+        Queue.enqueue_job(job_map, None) # passing no origin results in system origin
```

Comment:
Given this usage, what would your opinion be of making `origin` a named parameter, and always naming it when calling?

Reply:
It is more important we set and origin when we can, this is the only situation I am aware of that does not set an origin. I worry if the origin is optional, it will not be set when it should.

Reply:
Out of band: the `None` here is really the problem, this will be replaced with a system default origin to make the call clearer.